commands/build.go

```diff
@@ -113,16 +114,15 @@ func build(services *stack.Services, queueDepth int, shrinkwrap bool) {
 		go func(index int) {
 			wg.Add(1)
 			for function := range workChannel {
-				fmt.Printf("[%d] > Building: %s.\n", index, function.Name)
+				fmt.Printf(aec.YellowF.Apply("[%d] > Building %s.\n"), index, function.Name)
 				if len(function.Language) == 0 {
 					fmt.Println("Please provide a valid --lang or 'Dockerfile' for your function.")
-
 				} else {
 					builder.BuildImage(function.Image, function.Handler, function.Name, function.Language, nocache, squash, shrinkwrap)
 				}
+				fmt.Printf(aec.YellowF.Apply("[%d] < Building %s done.\n"), index, function.Name)
 			}
 
-			fmt.Printf("[%d] < Builder done.\n", index)
```

Comment:
Can we keep this and maybe say: Builder [%d] finished.

Reply:
`fmt.Printf(aec.YellowF.Apply("[%d] Builder finished.\n"), index)`
is it's ok with color and index as prefix ?